src/test/java/TopNavigationTest.java

```diff
@@ -0,0 +1,150 @@
+import components.TopNavigation;
+import enums.Attribute;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import utils.WebDriverUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TopNavigationTest {
+    private static final String HOMEPAGE_URL = "https://magento.softwaretestingboard.com/";
+    private static final By topNavigationBarCategoryNameFirstLevel = By.cssSelector("ul a > span:not(.ui-menu-icon)");
+    private static final By topNavigationBarCategoryFirstLevelLink = By.cssSelector("ul a");
+    private static final By topNavigationBarCategoryNameSecondLevelForWomen = By.cssSelector(".navigation li.level0:nth-child(2) ul.level0 li.level1:nth-child(1) .level2 a > span:not(.ui-menu-icon)");
+    private static final By topNavigationBarCategorySecondLevelForWomenLink = By.cssSelector(".navigation li.level0:nth-child(2) ul.level0 li.level1:nth-child(1) .level2 a");
+    private final By topNavigationBarSecondLevelForWomanTops = By.cssSelector(".navigation li.level0:nth-child(2) ul.level0 li.level1:nth-child(1)");
+
+    private WebDriver driver;
+    private TopNavigation topNavigation;
+    private List<WebElement> topCategoriesList;
+    private List<WebElement> secondLevelTopCategoriesList;
+
+    @BeforeClass
+    public void setUp() {
+        driver = WebDriverUtils.createWebDriver();
+        driver.get(HOMEPAGE_URL);
+        topNavigation = new TopNavigation(driver);
+        List<WebElement> topCategoriesList;
```

Comment:
List<WebElement> topCategoriesList is already declared as class level field.
Here in setUp() method topCategoriesList should be initialized.
topCategoriesList = topNavigation.getTopCategoriesList();